src/Network/Core.cpp

```diff
@@ -372,21 +372,19 @@ int Handle(EXCEPTION_POINTERS* ep) {
 
 [[noreturn]] void CoreNetwork() {
     while (true) {
-#if not defined(__MINGW32__)
+#if defined(_WIN32)
         __try {
-#endif
-
             CoreMain();
-
-#if not defined(__MINGW32__) and not defined(__linux__)
         } __except (Handle(GetExceptionInformation())) { }
-#elif not defined(__MINGW32__) and defined(__linux__)
-    }
-    catch (...) {
-        except("(Core) Code : " + std::string(strerror(errno)));
-    }
+#elif defined(__linux__) || defined(__APPLE__)
```

Comment:
Please make it that the launcher can still function if it was compiled with mingw, I'd just turn this line into an else

Reply:
I try to fix it